__tests__/app.test.js

```diff
@@ -126,4 +126,68 @@ describe("GET /api/articles/:article_id",()=>{
             expect(body.message).toBe("Article does not exist")
         })
     })
+})
+describe("GET /api/articles/:article_id/comments",() => {
+    test("GET: 200 sends an array of comment for the article_id",()=>{
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments.length).toBeGreaterThan(0)
+            const commentObject = {
+                'comment_id' : 1,
+                'votes' : 30,
+                'created_at' : '2018-05-30T15:59:13.341Z',
+                'body' : 'I like this fab comment',
+                'article_id' : 3
+            }
+            const desiredCommentObject = {
+                'comment_id' : 1,
+                'article_id' : 3,
+            }
+            body.comments.forEach((comment) => {
+                expect(typeof comment.comment_id).toBe('number')
+                expect(typeof comment.votes).toBe('number')
+                expect(typeof comment.created_at).toBe('string')
+                expect(typeof comment.author).toBe('string')
+                expect(typeof comment.body).toBe('string')
+                expect(typeof comment.article_id).toBe('number')
+                expect(commentObject).toMatchObject(desiredCommentObject)
```

Comment:
I'm not sure why you need this extra expect. You've already proven that the return value has the keys that you need.